bedrock/base/__init__.py

```diff
@@ -1,3 +1,6 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
+import markus
+
+metrics = markus.get_metrics()
```

Comment:
Out of curiosity, could this go in settings, just after markus is configured, or is there a risk that lazy-loading or similar will stop it being triggered?

Reply:
Possibly but I don't feel like it's a "setting", per se. But it can be used across the code which is why I put it in `base/`.